File: extensions/panache/hibernate-orm-panache-kotlin/deployment/src/test/java/io/quarkus/hibernate/orm/panache/kotlin/deployment/test/multiple_pu/second/SecondEntity.java
package io.quarkus.hibernate.orm.panache.kotlin.deployment.test.multiple_pu.second;

import javax.persistence.Entity;

import io.quarkus.hibernate.orm.panache.kotlin.PanacheEntity;

@Entity
public class SecondEntity extends PanacheEntity {

    public String name;
}

File: extensions/panache/hibernate-orm-panache-kotlin/deployment/src/test/kotlin/io/quarkus/hibernate/orm/panache/kotlin/deployment/test/multiple_pu/second/SecondEntity.kt
package io.quarkus.hibernate.orm.panache.kotlin.deployment.test.multiple_pu.second

import io.quarkus.hibernate.orm.panache.kotlin.PanacheCompanion
import io.quarkus.hibernate.orm.panache.kotlin.PanacheEntity
import javax.persistence.Entity

@Entity
class SecondEntity : PanacheEntity() {
    companion object: PanacheCompanion<SecondEntity>
    var name: String? = null
}
